### PR TITLE
add ToC of AdGuard Aff Program

### PR DESCRIPTION
**Pull request by el-termikael**

### CODE REVIEW

File: adguardpartner.com/terms-and-conditions.html.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+(c) To keep you informed about the Affiliate Account and the Affiliate Program, we will process your email address.
+(d) To provide you with our Support Service, AdGuard will process the information you provide when creating a support ticket. Please note that in most cases we do not need any additional personal data to provide you with support.
+
+### (3) Sharing your personal data with third parties
+
+To process payments via the payout method you have chosen, AdGuard will share some of your personal data with a third-party payment service that you have chosen to receive your payouts.
+
+(a) If you choose PayPal as your payout method, AdGuard will share the email address you indicate with PayPal (Europe) S.à r.l. et Cie, S.C.A. PayPal Privacy Policy: https://www.paypal.com/cy/webapps/mpp/ua/privacy-full.
+(b) If you choose Mellow as your payout method, AdGuard will share the email address you indicate with TMS SolarWeb Ltd. Mellow Privacy Policy: https://mellow.io/documents/privacy-policy.
+AdGuard does not request or have access to your accounts in third-party payment services.
```
Comment (ameshkov) [inline adguardpartner.com/terms-and-conditions.html.md:252]:
```suggestion

AdGuard does not request or have access to your accounts in third-party payment services.
```

**Review by ameshkov — CHANGES_REQUESTED** [review]
There are a lot of places where you need either to use list formatting or separate paragraphs because markdown will disregard new lines.

File: adguardpartner.com/terms-and-conditions.html.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+### (7) Period for bringing actions
+
+No action, regardless of form, arising out of the transactions under these Terms, may be brought by either party hereto more than one (1) year after the cause of action has accrued, except that an action for infringement of intellectual property rights may be brought within the maximum applicable statutory period.
+
+## 15. Contact information
+
+Should you have any questions concerning these Terms, or if you desire to contact AdGuard for any reason, please contact our Support Service:
+
+Email: support@adguard.com
+Website: https://aff.adguard.com/
```
Comment (ameshkov) [inline adguardpartner.com/terms-and-conditions.html.md:369]: adguardpartner
```suggestion
Website: https://adguardpartner.com/
```

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: adguardpartner.com/terms-and-conditions.html.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
+# Terms and Conditions of AdGuard Affiliate Program
+
+*February 26, 2024*
+
+## 1. General
+
+AdGuard Software Limited (hereafter, “AdGuard”, “we,” “us,” “our”) is a company registered in Limassol, Cyprus, registered office is at Anexartesias and Athinon 79, Nora Court Flat/Office 203–205, 3040 Limassol, Cyprus.
+
+AdGuard is the rights holder of AdGuard Ad Blocker: https://adguard.com/, AdGuard VPN: https://adguard-vpn.com/, and AdGuard DNS: https://adguard-dns.io/ (hereafter, collectively, the “Software”). We also operate the websites of the Software mentioned above (hereafter, the “Software Websites”), as well as the AdGuard Affiliate Account service (hereafter, the “Affiliate Account”) available at https://aff.adguard.com/ (hereafter, the “Affiliate Website”).
```
Comment (bniwredyc) [inline adguardpartner.com/terms-and-conditions.html.md:9]: Let's make all links pretty like ```[adguard.com](https://adguard.com/)```

File: adguardpartner.com/terms-and-conditions.html.md [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+
+### (7) Period for bringing actions
+
+No action, regardless of form, arising out of the transactions under these Terms, may be brought by either party hereto more than one (1) year after the cause of action has accrued, except that an action for infringement of intellectual property rights may be brought within the maximum applicable statutory period.
+
+## 15. Contact information
+
+Should you have any questions concerning these Terms, or if you desire to contact AdGuard for any reason, please contact our Support Service:
+
+Email: support@adguard.com
```
Comment (bniwredyc) [inline adguardpartner.com/terms-and-conditions.html.md:368]: In this case ```[support@adguard.com](mailto:support@adguard.com)```

File: adguardpartner.com/terms-and-conditions.html.md [hunk 5]
```diff
@@ -0,0 +1,372 @@
+# Terms and Conditions of AdGuard Affiliate Program
+
+*February 26, 2024*
+
+## 1. General
+
+AdGuard Software Limited (hereafter, “AdGuard”, “we,” “us,” “our”) is a company registered in Limassol, Cyprus, registered office is at Anexartesias and Athinon 79, Nora Court Flat/Office 203–205, 3040 Limassol, Cyprus.
+
+AdGuard is the rights holder of AdGuard Ad Blocker: [adguard.com](https://adguard.com/), AdGuard VPN: [adguard-vpn.com](https://adguard-vpn.com/), and AdGuard DNS: [adguard-dns.io](https://adguard-dns.io/) (hereafter, collectively, the “Software”). We also operate the websites of the Software mentioned above (hereafter, the “Software Websites”), as well as the AdGuard Affiliate Account service (hereafter, the “Affiliate Account”) available at [aff.adguard.com](https://aff.adguard.com/) (hereafter, the “Affiliate Website”).
```
Comment (ameshkov) [inline adguardpartner.com/terms-and-conditions.html.md:9]:
```suggestion
AdGuard is the rights holder of AdGuard Ad Blocker: [adguard.com](https://adguard.com/), AdGuard VPN: [adguard-vpn.com](https://adguard-vpn.com/), and AdGuard DNS: [adguard-dns.io](https://adguard-dns.io/) (hereafter, collectively, the “Software”). We also operate the websites of the Software mentioned above (hereafter, the “Software Websites”), as well as the AdGuard Affiliate Account service (hereafter, the “Affiliate Account”) available at [adguardpartner.com](https://adguardpartner.com/) (hereafter, the “Affiliate Website”).
```

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: adguardpartner.com/terms-and-conditions.html.md [hunk 6]
```diff
@@ -0,0 +1,372 @@
...
+    a. A user downloads AdGuard Ad Blocker from the Software Website accessed by the user via your personalized link or through your personalized installation link,
+    b. The downloaded Software is installed, and
+    c. The installed Software is activated by the purchased subscription. The subscription does not have to be purchased through your personalized link. See Restrictions below for additional conditions on the purchased subscription.
+- (b) You will earn your commission for a subscription renewal when the subscription previously activated in the installed Software that was downloaded via your personalized link or through your personalized installation link is renewed.
+
+### (3) Your commissions for AdGuard VPN
+
+You are entitled to receive forty percent (40%) of the purchase or renewal of subscriptions to the AdGuard VPN made by users referred by you, subject to the following conditions:
+
+- (a) You earn your commission for a subscription purchase when
```
Comment (ameshkov) [inline adguardpartner.com/terms-and-conditions.html.md:68]: Fix formatting here
image

File: adguardpartner.com/terms-and-conditions.html.md [hunk 7]
```diff
@@ -0,0 +1,372 @@
...
+- (f) YOU ARE RESPONSIBLE FOR ENSURING THAT YOUR PARTICULAR USE OF THE AFFILIATE ACCOUNT SERVICE DOES NOT VIOLATE ANY APPLICABLE LAW, THE RIGHTS OF ANY THIRD PARTY, OR YOUR CONTRACTUAL OBLIGATIONS TO ANY THIRD PARTY.
+
+## 12. Limitation of liability
+
+(a) TO THE MAXIMUM EXTENT PERMITTED BY LAW, NEITHER ADGUARD NOR ITS SUPPLIERS OR AFFILIATES SHALL BE LIABLE TO YOU OR ANY THIRD PARTY FOR ANY INDIRECT, SPECIAL, INCIDENTAL, PUNITIVE, COVER OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, DAMAGES FOR THE INABILITY TO USE EQUIPMENT OR ACCESS DATA, LOSS OF DATA, LOSS OF BUSINESS, LOSS OF PROFITS, BUSINESS INTERRUPTION OR THE LIKE), ARISING OUT OF THE USE OF, OR INABILITY TO USE THE AFFILIATE ACCOUNT SERVICE OR THE AFFILIATE WEBSITE PROVIDED UNDER THESE TERMS WHETHER BASED ON ANY THEORY OF LIABILITY, INCLUDING BREACH OF CONTRACT, BREACH OF WARRANTY, TORT (INCLUDING NEGLIGENCE), PRODUCT LIABILITY OR OTHERWISE, EVEN IF ADGUARD OR ITS SUPPLIERS OR AFFILIATES HAVE BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES AND EVEN IF A REMEDY SET FORTH HEREIN IS FOUND TO HAVE FAILED OF ITS ESSENTIAL PURPOSE.
```
Comment (ameshkov) [inline adguardpartner.com/terms-and-conditions.html.md:308]: Use list formatting here

**Review by ameshkov — CHANGES_REQUESTED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**